conclave-enclave/src/main/kotlin/com/r3/conclave/enclave/internal/GramineEntryPoint.kt

```diff
@@ -41,20 +45,21 @@ object GramineEntryPoint {
     private fun initialiseEnclave(
         enclaveClassName: String,
         hostInterface: SocketEnclaveHostInterface,
-        pythonScript: Path?
+        pythonScript: Path?,
+        enclaveMode: EnclaveMode
     ) {
         val enclaveClass = Class.forName(enclaveClassName)
         val enclave = enclaveClass.asSubclass(Enclave::class.java)
-                .getDeclaredConstructor()
-                .apply { isAccessible = true }
-                .newInstance()
+            .getDeclaredConstructor()
+            .apply { isAccessible = true }
+            .newInstance()
```

Comment:
It's a Java thing I think. The purpose of it is to make it easy to distinguish between normal indentation and wrapping that has been done to preserve horizontal space. See [here](https://www.oracle.com/java/technologies/javase/codeconventions-indentation.html), specifically the bullet point "If the above rules lead to confusing code or to code that's squished up against the right margin, just indent 8 spaces instead". I think most editors just default to eight spaces all the time these days. Additionally, you'll find if you put a newline in the middle of something like a parameter list, you'll notice that intellij will automatically double indent rather than single.

Reply:
That is my point: my editor does not default to that :-)! I see what you mean and I would be fine with that, but I think we should establish a common convention, with convention file included. Plus a TC job complaining about indentation rules.
Also noticing that Shams in his Plugin cleanup is converting from 8 to 4 spaces!
https://github.com/R3Conclave/conclave-core-sdk/pull/90/files#diff-41d01a82255cd13f751ee17782380ef3a1b5c17095358ac70ab00acb6fc557e6R26